src/providers/VernaProvider.tsx

```diff
@@ -92,18 +100,20 @@ interface VernaContextProviderProps {
   objectFieldTemplate: ObjectFieldTemplateType;
   defaultSelector?: string;
   isEditor: boolean;
+  transformErrors?: (errors: AjvError[]) => AjvError[];
 }
 
 function VernaProvider({
   configSchema,
+  children,
   defaultSchema,
   defaultUiSchema,
   defaultFormValues,
   defaultWidgets,
   objectFieldTemplate,
   defaultSelector,
   isEditor,
-  children,
+  transformErrors,
 }: PropsWithChildren<VernaContextProviderProps>) {
   // The full schemas are not used by the lib itself but may be used for further implementation
```

Comment:
```suggestion
  // The full schema is not used by the lib itself but may be used for further implementation
```

Reply:
I updated it for:
-  // The full schemas are not used by the lib itself but may be used for further implementation
+  // Both fullSchema & fullUiSchema are not used by the lib itself but may be
+  // used for further implementation